sourcetracker/tests/test_sourcetracker.py

```diff
@@ -10,119 +10,226 @@
-                                                           biom_table, sort)
-        exp_envs = np.array(['1', '2'])
-        exp_collapsed_sources = \
-            np.array([[0, 5, 10, 15, 20, 25, 30, 35, 40, 45],
-                      [3, 13, 23, 33, 43, 53, 63, 73, 83, 93]])
-        np.testing.assert_array_equal(obs_collapsed_sources,
-                                      exp_collapsed_sources)
-        np.testing.assert_array_equal(obs_envs, exp_envs)
-
-        category = 'cat1'
-        samples = ['s4', 's2', 's0', 's1']
-        sort = True
-        obs_envs, obs_collapsed_sources = collapse_sources(samples,
-                                                           sample_metadata,
-                                                           category,
-                                                           biom_table, sort)
-        exp_envs = np.array(['1', '2'])
-        exp_collapsed_sources = \
-            np.array([[4, 14, 24, 34, 44, 54, 64, 74, 84, 94],
-                      [3, 13, 23, 33, 43, 53, 63, 73, 83, 93]])
-        np.testing.assert_array_equal(obs_collapsed_sources,
-                                      exp_collapsed_sources)
-        np.testing.assert_array_equal(obs_envs, exp_envs)
-
-        category = 'cat2'
-        samples = ['s4', 's2', 's0', 's1']
-        sort = True
-        obs_envs, obs_collapsed_sources = collapse_sources(samples,
-                                                           sample_metadata,
-                                                           category,
-                                                           biom_table, sort)
-        exp_envs = np.array(['B', 'C', 'D'])
-        exp_collapsed_sources = \
-            np.array([[0, 5, 10, 15, 20, 25, 30, 35, 40, 45],
-                      [1, 6, 11, 16, 21, 26, 31, 36, 41, 46],
-                      [6, 16, 26, 36, 46, 56, 66, 76, 86, 96]])
-        np.testing.assert_array_equal(obs_collapsed_sources,
-                                      exp_collapsed_sources)
-        np.testing.assert_array_equal(obs_envs, exp_envs)
+from sourcetracker._sourcetracker import (biom_to_df,
+                                          intersect_and_sort_samples,
+                                          collapse_source_data,
+                                          subsample_dataframe,
+                                          check_and_correct_data,
+                                          collate_gibbs_results,
+                                          generate_taxon_sequence, get_samples,
+                                          generate_environment_assignments,
+                                          cumulative_proportions,
+                                          single_sink_feature_table,
+                                          ConditionalProbability,
+                                          gibbs_sampler)
+
+
+class test_check_and_correct_data(TestCase):
```

Comment:
By convention, class names should be in camel case. I'd recommend making this change as it's a widely used convention.